src/pages/index.js

```diff
@@ -116,15 +114,30 @@ function renderCard(cardData) {
             cardDeleteModal.renderLoading(false);
           }); 
       });
-      card.processLikeClick();
+      
     },
-   
-
+    processLikeClick,
     userId,
   });
 
   section.addItem(card.getCard());
 }
+function processLikeClick(card) {
+  const isLiked = card.isLiked();
+  if (isLiked) {
+    () => {
+      this.setLikesInfo(data.likes);
+    }
```

Comment:
```js

    () => {
      this.setLikesInfo(data.likes);
    }

```


Please, delete it because it has not effect here

Reply:
deleted.